### ISSUE TITLE
focus repositioning after Esc key

### ISSUE THREAD

**Issue by renow-luxembourg** [Bluefieldscom/intl-tel-input#2104]
First of all, I would like to thank you for this exceptional widget.

During an accessibility audit of one of our sites, a focus repositioning issue was identified. 
Specifically, when navigating using the keyboard, if the list of countries is expanded and the Esc key is pressed, the focus is lost. The focus should be moved to the element that open the country list.

The component have to respect the RGAA/RAWeb accessibility guidelines for French-speaking countries. The test that failed is 12.8 about navigation sequence consistency (https://accessibilite.public.lu/en/raweb1/criteres.html#crit-12-8)

Note : We are currently use the version 17.0.12 of intl-tel-input but the issue is still visible on the demo page of https://intl-tel-input.com/

**Comment by jackocnr** [comment]
Thanks for reporting this. And thanks for the PR as well. In the end, I decided to implement the fix in a different way - so it only triggers a focus in this specific case (Escape key), and it focuses the select-country-button instead, which is more like how native select elements work. Thanks again.
